cmd/alertmanager/main.go

```diff
@@ -180,10 +182,14 @@ func run() int {
 	}
 
 	var (
-		configFile      = kingpin.Flag("config.file", "Alertmanager configuration file name.").Default("alertmanager.yml").String()
-		dataDir         = kingpin.Flag("storage.path", "Base path for data storage.").Default("data/").String()
-		retention       = kingpin.Flag("data.retention", "How long to keep data for.").Default("120h").Duration()
-		alertGCInterval = kingpin.Flag("alerts.gc-interval", "Interval between alert GC.").Default("30m").Duration()
+		configFile = kingpin.Flag("config.file", "Alertmanager configuration file name.").Default("alertmanager.yml").String()
+		dataDir    = kingpin.Flag("storage.path", "Base path for data storage.").Default("data/").String()
+		retention  = kingpin.Flag("data.retention", "How long to keep data for.").Default("120h").Duration()
+
```

Comment:
MAY: leave 183-186 alone and add your new stuff after a blank line to reduce lines diff.

Reply:
Done.